src/client/src/OpenFin/Window/WindowControls.tsx

```diff
@@ -34,7 +38,14 @@ export const WindowControls = ({ close, minimize, maximize, popIn }: Props) => {
   }, [])
 
   async function wrappedClose() {
-    if (inMainOpenFinWindow()) {
+    if (inMainReactiveTraderWindow()) {
+      // ONLY if main win is primary win of platform, close all platform windows
+      if (isWindowPlatformPrimary()) {
+        console.warn(`AJG: close platform`)
```

Comment:
do we intend to merge with the console.warn?

Reply:
we do not 😁